device-base/Dockerfile.i386.edison.tpl

```diff
@@ -16,28 +16,48 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 # MRAA
 ENV MRAA_COMMIT 887acf54e1f674ad2acb506c6e6b982d6074c73e
 ENV MRAA_VERSION 1.0.0
+# UPM
+ENV UPM_COMMIT 29bfa7ef79ec7e944e8803b39fe1fb71ab7c0825
+ENV UPM_VERSION 0.7.0
 
 # Install mraa
 RUN set -x \
 	&& buildDeps=' \
 		build-essential \
-		cmake \
 		git-core \
 		libpcre3-dev \
 		python-dev \
 		swig \
+		pkg-config \
+		curl \
 	' \
 	&& apt-get update && apt-get install -y $buildDeps --no-install-recommends && rm -rf /var/lib/apt/lists/* \
+	&& mkdir /cmake \ 
+	&& curl -SL https://cmake.org/files/v3.5/cmake-3.5.2.tar.gz -o cmake.tar.gz \
+	&& echo "92d8410d3d981bb881dfff2aed466da55a58d34c7390d50449aa59b32bb5e62a  cmake.tar.gz" | sha256sum -c - \
```

Comment:
cmake on wheezy is too old while UPM requires newer cmake version so we need to install from source @petrosagg